content/en/docs/tasks/administer-cluster/highly-available-master.md

```diff
@@ -170,6 +170,28 @@ to access a replica via its ephemeral public IP, you must skip TLS verification.
 To allow etcd clustering, ports needed to communicate between etcd instances will be opened (for inside cluster communication).
 To make such deployment secure, communication between etcd instances is authorized using SSL.
 
+### API server identity
+
+{{< feature-state state="alpha"  for_k8s_version="v1.20" >}}
+
+The API Server Identity feature is controlled by a feature gate
+and is not enabled by default.  See
+[Feature Gates](/docs/reference/command-line-tools-reference/feature-gates/)
+for a general explanation of feature gates and how to enable and disable them.  The
+name of the feature gate is "APIServerIdentity".  You can enable API Server Identity
+by adding the following command-line flag to your `kube-apiserver` invocation:
```

Comment:
I'd write this as:
```suggestion
The API Server Identity feature is controlled by a
[feature gate](/docs/reference/command-line-tools-reference/feature-gates/)
and is not enabled by default. You can activate API Server Identity by enabling
the feature gate named `APIServerIdentity` when you start the
{{< glossary_tooltip text="API Server" term_id="kube-apiserver" >}}:
```

Reply:
or:
```suggestion
_API Server identity_ is behind a
[feature gate](/docs/reference/command-line-tools-reference/feature-gates/)
named `APIServerIdentity`, that is not enabled by default.
You can enable API Server Identity by adding the following command-line
flag to your `kube-apiserver` invocation:
```

Reply:
Thanks. Updated the paragraph with the first suggestion.